sklearn/kernel_approximation.py

```diff
@@ -1,521 +1,638 @@
-"""
```

Comment:
not too sure why these diffs are massive, but it makes it very hard to review (since one can't tell what changes you made)...